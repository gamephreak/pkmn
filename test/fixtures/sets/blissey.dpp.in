Blissey @ Leftovers  
Ability: Natural Cure
EVs: 148 HP / 252 Def / 108 SpD  
Bold Nature  
- Wish  
- Toxic  
- Protect  
- Seismic Toss  
