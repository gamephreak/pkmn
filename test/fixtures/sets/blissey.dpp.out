Blissey @ Leftovers  
EVs: 148 HP / 252 Def / 108 SpD  
Bold Nature  
- Wish  
- Toxic  
- Protect  
- Seismic Toss  

